app/config/packages/nelmio_cors.yaml

```diff
@@ -0,0 +1,10 @@
+nelmio_cors:
```

Comment:
You have to use a plugin to manage cors?

Reply:
Easier than doing it by hand. The header must be returned first on the OPTION request. The lib do it automatically for me.